components/function-controller/config/config.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+items:
+  - apiVersion: v1
+    data:
+      Dockerfile: |-
+        FROM kubeless/nodejs@sha256:5c3c21cf29231f25a0d7d2669c6f18c686894bf44e975fcbbbb420c6d045f7e7
+        USER root
+        RUN export KUBELESS_INSTALL_VOLUME='/kubeless' && \
+            mkdir /kubeless && \
+            cp /src/handler.js /kubeless && \
+            cp /src/package.json /kubeless && \
+            /kubeless-npm-install.sh
+        USER 1000
+    kind: ConfigMap
+    metadata:
+      annotations:
+      labels:
+        function: serverless-function-controller
+      name: dockerfile-nodejs-6
+      namespace: serverless-system
+  - apiVersion: v1
+    data:
+      Dockerfile: |-
+        FROM kubeless/nodejs@sha256:5c3c21cf29231f25a0d7d2669c6f18c686894bf44e975fcbbbb420c6d045f7e7
+        USER root
+        RUN export KUBELESS_INSTALL_VOLUME='/kubeless' && \
+            mkdir /kubeless && \
+            cp /src/handler.js /kubeless && \
+            cp /src/package.json /kubeless && \
+            /kubeless-npm-install.sh
+        USER 1000
+    kind: ConfigMap
+    metadata:
+      labels:
+        function: serverless-function-controller
+      name: dockerfile-nodejs-8
+      namespace: serverless-system
+kind: List
```

Comment:
Let's avoid Lists inside manifests, Lists are for API responses.

Reply:
fixed.